src/picongpu/include/plugins/radiation/Radiation.hpp

```diff
@@ -606,6 +610,7 @@ class Radiation : public ISimulationPlugin
       splash::DataCollector::initFileCreationAttr(fAttr);
 
       fAttr.fileAccType = splash::DataCollector::FAT_READ;
+      fAttr.enableCompression = compressionOn;
```

Comment:
There is no need to set compression on read, this is transparent for the user.
Please remove this line.